src/index.js

```diff
@@ -0,0 +1,33 @@
+import App from './components/App';
+import { AppContainer } from 'react-hot-loader';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+let isOfficeInitialized = false;
+
+const title = 'Accord Project';
+
+const render = Component => {
+  ReactDOM.render(
+    <AppContainer>
+      <Component title={title} isOfficeInitialized={isOfficeInitialized} />
```

Comment:
We could just keep it `isOfficeInitialized={false}` for now, right?

Reply:
I think we want to keep it the way @algomaster99 has it, as `let isOfficeInitialized = false` on line 6 so that it can be reassigned on line 21. Otherwise, written as `isOfficeInitialized={false}`, the value will never update to `true` and we'll always end up in the condition on line 23 in `App.js`.

Reply:
AH! I missed L21 thanks